01_install_requirements.sh

```diff
@@ -13,10 +13,7 @@ if [ -z "${METAL3_DEV_ENV}" ]; then
   sync_repo_and_patch metal3-dev-env https://github.com/metal3-io/metal3-dev-env.git
   METAL3_DEV_ENV="${REPO_PATH}/metal3-dev-env/"
 fi
-
 pushd ${METAL3_DEV_ENV}
-# FIXME: Pin to aafbb89df8df88bc3ecbfe59592149c929432515
-git reset aafbb89df8df88bc3ecbfe59592149c929432515 --hard
```

Comment:
I think I'd prefer to keep the pin, but on something newer. Eventually we could automate pushing it forward if OpenShift CI passes.

We need to prove the job's stability and this breakage proves using metal3-dev-env latest is always going to be risky.

Reply:
OK, I'll add this back.